supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -128,4 +128,30 @@ Avoid long introductions and unnecessary context. Shorten unnecessarily long sen
 === Principle 5: Error recovery, verification, and troubleshooting
 Recognize that people make mistakes and need to verify that they have completed a task. Be sure to include troubleshooting, error recovery, and verification steps.
 
+[[users]]
+== Users
+In most cases, the word "user" refers to a person or a person's user account, and therefore would be considered animate. In these cases, use animate personal pronouns such as "who".
```

Comment:
Sorry, I'm not sure I understand what you mean here.

Do you mean that we sometimes refer to "users" as those who perform the tasks (i.e. reader)?

And Linux and SELinux users and similar would be role references?

Reply:
I was trying to suggest some direct guidelines that would help writers make a decision. I was thinking something on the lines of:
"Task performers" -> Use animate
"Role references"-> Use inanimate

Reply:
I've thought about this, but I think this is too simple a recommendation. We can refer to personal users even when they are not task performers. In fact, we should typically refer to task performers as "you" and write directly to them.

Also, we should primarily write around role references instead of using the inanimate pronouns.

So I think the current recommendation is more accurate:

> In these specific cases, and only if you cannot write around it, you can refer to these inanimate users with inanimate personal pronouns such as "that".

Reply:
sounds good!